test/features/page_objects/poa.rb

```diff
@@ -0,0 +1,142 @@
+require_relative '../utils/index.rb'
+
+class POA
+
+  def initialize(driver)
+    @driver = driver
+  end
+
+  def file_upload
+    element = @driver.find_element(:css, '.onfido-sdk-ui-Uploader-uploadArea input[type="file"]')
+    @driver.execute_script("return arguments[0].setAttribute('style','display: true');", element)
+    @driver.execute_script("return arguments[0].value = '';", element) unless element.attribute('value').empty?
+    element
+  end
+
+  def uploaded_image
+    @driver.find_element(:css, '.onfido-sdk-ui-Confirm-CaptureViewer-image')
+  end
+
+  def verify_uk_address_title
+    @driver.find_element(:xpath, "//span[contains(.,'Let’s verify your UK address')]")
+  end
+
+  def need_a_doc_text
+    @driver.find_element(:xpath, "//p[contains(.,'You’ll need a document that:')]")
+  end
+
+  def poa_requirements
+    @driver.find_element(:xpath, "//span[contains(.,'Shows your current address')]")
+    @driver.find_element(:xpath, "//span[contains(.,'Matches the address you used on signup')]")
+    @driver.find_element(:xpath, "//span[contains(.,'Is your most recent document')]")
+  end
+
+  def start_verification_button
+    @driver.find_element(:xpath, "//button[contains(.,'Start verification')]")
+  end
+
+  ### Document selection screen elements
+  def document_selection_screen_title
+    @driver.find_element(:xpath, "//span[@class='onfido-sdk-ui-Title-titleSpan'][contains(.,'Select a UK document')]")
+  end
+
+  def document_selection_screen_subtitle
+    @driver.find_element(:xpath, "(//div[contains(.,'These are the documents most likely to show your current home address')])[9]")
+  end
+
+  def bank_building_statement_cell
+    @driver.find_element(:xpath, "//div[@class='onfido-sdk-ui-DocumentSelector-option'][contains(.,'Bank/Building Society Statemente-statements accepted')]")
+  end
+
+  def bank_building_statement_icon
+    @driver.find_element(:xpath, "//div[contains(@class,'onfido-sdk-ui-DocumentSelector-icon onfido-sdk-ui-DocumentSelector-icon-bank-building-society-statement')]")
+  end
+
+  def bank_building_statement_title
+    @driver.find_element(:xpath, "//p[@class='onfido-sdk-ui-DocumentSelector-label'][contains(.,'Bank/Building Society Statement')]")
```

Comment:
The same approach as in `document_upload_intro_screen(title)` can be used here and you will be able to remove all the redundant methods